todo_app/app.py

```diff
@@ -1,12 +1,11 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.session_items import add_item, get_items
+from todo_app.data.trello_items import add_item, get_items
```

Comment:
It doesn't show any `add_item` function in the `trello_items.py` file currently - was there something else to push to this branch maybe?